lib/icalendar/calendar.rb

```diff
@@ -5,6 +5,17 @@ class Calendar < Component
     required_property :prodid
     optional_single_property :calscale
     optional_single_property :ip_method
+    optional_single_property :ip_name
```

Comment:
Several of these properties can actually be defined more than once, so they should use the `optional_property` method instead of `optional_single_property` On the calendar component, they are `name`, `description`, `categories`, and `image`

```suggestion
    optional_property :ip_name
```